05-specifying-sample-designs.Rmd

```diff
@@ -1 +1,705 @@
+
+$$se(\bar{y})=\sqrt{\frac{s^2}{n}\left( 1-\frac{n}{N} \right)}$$ and
+
+$$s^2=\frac{1}{n-1}\sum_{i=1}^n\left(y_i-\bar{y}\right)^2.$$
+
+This standard error estimate might look very similar to equations in other applications seen except for the bit on the right side of the equation $1-\frac{n}{N}$ - this is call the finite population correction factor (FPC). If the size of the frame, $N$, is very large, the FPC is negligible so it is often ignored. 
+
+To estimate proportions, we define $x_i$ as the indicator that the outcome is observed. That is, $x_i=1$ if the outcome is observed and $x_i=0$ if the outcome is not observed. Then the estimated proportion from a SRS design is:
+
+$$\hat{p}=\frac{1}{n}\sum_{i=1}^n x_i $$
+and the estimated standard error of the proportion is:
+
+$$se(\hat{p})=\sqrt{\frac{\hat{p}(1-\hat{p})}{n-1}\left(1-\frac{n}{N}\right)} $$
+
+#### The syntax {-} 
+
+If a sample that was drawn through SRS and had no nonresponse or other weighting adjustments, in R specify this design as:
+
+```{r}
+#| label: srs1
+#| eval: FALSE
+des_srs1 <- dat %>%
+  as_survey_design(fpc = fpcvar)
+```
+
+where `dat` is a tibble or data.frame with the survey data and `fpcvar` is a variable on the tibble which indicates the size of the sampling frame. If the frame was very large, sometimes the frame size is not provided. In that case, specify the design as:
+
+```{r}
+#| label: srs2
+#| eval: FALSE
+des_srs2 <- dat %>%
+  as_survey_design()
+```
+
+If some post-survey adjustments were implemented and the weights are not all equal, specify the design as:
+
+```{r}
+#| label: srs3
+#| eval: FALSE
+des_srs3 <- dat %>%
+  as_survey_design(weights = wtvar, fpc = fpcvar)
+```
+
+where `wtvar` is the variable for the weight on the data. Again, the fpc can be omitted if it is not necessary because the frame is large.
+
+#### Example {-} 
+
+The {survey} package in R provides some example datasets to use and those will be used throughout this chapter. The Academic Performance Index (API) was a program administered by the California Department of Education. These datasets are derived from that data and include a population file (frame) of all schools with at least 100 students and several samples from that data. A SRS of 200 schools was sampled and included in `apisrs`. The SRS sample data is illustrated below:
+
+```{r}
+#| label: apisrs_display
+options(tidyverse.quiet = TRUE)
+library(tidyverse)
+library(survey)
+library(srvyr)
+
+data(api)
+
+apisrs_slim <- 
+  apisrs %>%
+  as_tibble() %>%
+  arrange(dnum, snum) %>%
+  select(cds, dnum, snum, dname, sname, fpc, pw)
+
+```
+
+School districts have identifiers `dnum` within counties and schools have identifiers of `snum` within districts along with their names, `dname` and `sname`. The unique identifier for a school is `cds` which is unique within the state. Additionally, the `fpc` is included as `fpc` and the weight as `pw`. This design should be specified as:
```

Comment:
Should we start this paragraph by stating something like "Reading the documentation about this dataset provides detail on the variables."  This way we can remind the reader that we don't just know this, but it is available for review.  I think Chapter 3 will cover this, but always good to have a refresher and show it in action.